client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -504,8 +504,7 @@ export default function CampaignItemDetails( props: Props ) {
 								</div>
 							</div>
 						</div>
-
-						{ status === 'completed' && (
+						{ ( status === 'completed' || status === 'finished' ) && (
```

Comment:
I'm guessing the 'completed' status doesn't exist. It should be just 'finished`. but I guess other status might want to see this information.

Campaign cancelled: User might have been billed when they cancel the campaign if some impressions got delivered

so I'm guessing it should be:

```
status === 'finished' || status === 'canceled'
```

Reply:
fixed.. please check and test it again...